man/docker-run.1.md

```diff
@@ -435,7 +435,10 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Docker to label the content with a private unshared label.
 Only the current container can use a private volume.
 
-Note: Multiple Volume options can be added separated by a ","
+Note:
```

Comment:
Please use the same next here as above.  >

The `host-dir` can be relative or absolute. This means both `/foo` or `foo` can be a `host-dir` value. The `container-dir` must always be an absolute path such as `/src/docs`.